src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -55,45 +60,49 @@ List<Message> messagesByUser = (List<Message>) request.getAttribute("messagesByU
     <% if(request.getSession().getAttribute("user") != null){ %>
       <a>Hello <%=request.getSession().getAttribute("user")%>!</a>
       <a href="/users/<%=request.getSession().getAttribute("user")%>">My Profile</a>
+      <a href="/logout.jsp">Logout</a>
     <% } else { %>
       <a href="/login">Login</a>
     <% } %>
     <a href="/about.jsp">About</a>
   </nav>
 
-    <div id="container">
+      <div id="container">
 
-      <% if(request.getAttribute("error") != null){ %>
-        <h2 style="color:red"><%= request.getAttribute("error") %></h2>
-      <% } %>
+        <% if(request.getAttribute("error") != null){ %>
+          <h2 style="color:red"><%= request.getAttribute("error") %></h2>
+        <% } %>
+
+        <% if(request.getSession().getAttribute("user") != null){ %>
+          <h1><%=request.getSession().getAttribute("user")%>'s Profile Page</h1>
+        <hr>
+          <strong>About <%=request.getSession().getAttribute("user")%></strong><br>
+          <p><%=user.getAboutMe()%></p>
+          <form action="/users/<%=request.getSession().getAttribute("user") %>" method="POST">
+
+        <% } %>
```

Comment:
I feel like you might want this closing brace down below the Sent Messages stuff so that the About Me box and the Sent Messages display don't show up if there is no user logged in.